### ISSUE TITLE
is this any chance related to the l2reborn server data?

### ISSUE THREAD

**Issue by whoizz** [danielcrane/l2reborn#1]
wondering what's the source of your data, in this repo, since their website data is VERY inaccurate - it does look to me that the data used by server comes from https://bitbucket.org/l2jserver/l2j-server-datapack/raw/38a842acaaba51a9aec23249e4f62bede0a1ad33/src/main/resources/sql/npc.sql - I've been trying to utilize the l2reborn website data (NPC Data) to find a good spot to solo as Cleric but so far every attempt to find a good NPC resulted in failures (meaning, the reported values in l2reborn website does not match the values used by the game server)

**Comment by danielcrane** [comment]
Hi Daniel,

This was the same data l2reborn was using at some point, but it has since
changed.

I have been working on updating my database search (which can be found
here: https://danielcrane.github.io/rebornfun/search.html) to include the
latest data, so it should be fully up-to-date in terms of drops and spoils
now. Mob locations may be a bit off (but probably largely similar), since
I've not yet updated that.

Feel free to PM me on Discord @FatChocobo if you have any questions.

**Comment by whoizz** [comment]
yeah, what I don't understand is how you sourced the data - it does look like you decoded some of the .dat files available in the L2Reborn modified client, however, in order to achieve any kind of "accuracy" you'd need the source .SQL or at least a current .CSV dump of the tables they utilize.. - I was looking into this because it takes hours and hours to select one spot to farm, and as much as I search it's inconsistency after inconsistency, be it in XP values, HP values, I even found issues with skill data (Resistances vs Weak Points).

as I mentioned before, most of this data is available out of Interlude branch, and I've seen cases it does not match 1-1 with my checkout files, so I'm wondering if you are using the Game Server values, from the database layer itself that is current "Live" (ie: you have achieve sync so if they modify you modify as well)

**Comment by danielcrane** [comment]
@whoizz For some reason I didn't get a notification about your reply, so I'm sorry that my response is so extremely late.

I was using game server files, for most things, yes. I was only using the game `.dat` files to get some things that aren't covered in those files (skill descriptions and some other things), and some of my scripts were made to modify the `.dat` files to add drop/spoil data.
